src/DI.Specification.Tests/DI.Specification.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <Description>Suite of xUnit.net tests to check for container compatibility with Microsoft.Extensions.DependencyInjection.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Extensions.DependencyInjection.Specification.Tests</AssemblyName>
+    <RootNamespace>Microsoft.Extensions.DependencyInjection.Specification</RootNamespace>
```

Comment:
Don't we not want namespaces in test repos to include `*.Tests`? Or is that not what this property means?